specification/synapse/resource-manager/Microsoft.Synapse/stable/2020-12-01/sqlPool.json

```diff
@@ -459,6 +459,63 @@
         }
       }
     },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Synapse/workspaces/{workspaceName}/sqlPools/{sqlPoolName}/move": {
```

Comment:
Move action should follow the guidelines as specified here: https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/resource-api-reference.md#move-resource

Reply:
This is also a proxy API that we're referring the SQL Swagger as-is. 


https://github.com/Azure/azure-rest-api-specs/blob/master/specification/sql/resource-manager/Microsoft.Sql/preview/2020-08-01-preview/Databases.json


This is the API that we invoke directly from our API so we do not change the contract that is already exposed.